cpp/include/cuvs/neighbors/cagra.hpp

```diff
@@ -519,6 +570,10 @@ struct index : cuvs::neighbors::index {
   raft::device_matrix<IdxT, int64_t, raft::row_major> graph_;
   raft::device_matrix_view<const IdxT, int64_t, raft::row_major> graph_view_;
   std::unique_ptr<neighbors::dataset<dataset_index_type>> dataset_;
+  // only float distances supported at the moment
+  std::optional<raft::device_vector<float, int64_t>> dataset_norms_;
```

Comment:
This is not saved during serialization. Do I understand correctly, that this will be recomputed when the index is created after loading the data?